tests/app/v2/template/test_get_template.py

```diff
@@ -37,6 +41,16 @@ def test_get_template_by_id_returns_200(
 
     json_response = json.loads(response.get_data(as_text=True))
```

Comment:
You didn't write this, but . . .
```suggestion
    json_response = response.get_json()
```